### ISSUE TITLE
Bug, whole scene is scaled by 2 on IOS when safari goes to background and opened again

### ISSUE THREAD

**Issue by peciMipumi** [mrdoob/three.js#9567]
I could reproduce it on the samples of threejs.org. 
Just open the sample on ios. (ipad or iphone). Close the browser, open another app. then open safari again and you will see that everything is rendered twice as big, and also moved to the right bottom of the page.

could it have somthing todo with the pixelratio (on those devices it is 2).
also note that the webglcontextlost event is not triggered.

OS:
IOS

**Comment by mrdoob** [comment]
Yeah, I have seen this behaviour. It doesn't happen in any other platform so it's a iOS bug. Hopefully they fixed this on iOS10!

**Comment by peciMipumi** [comment]
this only happens when the resize is being called. This is why the bug does not happen on the three.js example page, and happens when you open one of the examples on a new tab.

I tried the lessons on webglfundamentials (http://webglfundamentals.org/webgl/lessons/webgl-resizing-the-canvas.html) and this worked. When looking at the code the difference is that instead of doing the resize functionality on the resize event, they are checking if the size has changed every frame, and doing the resize just before the frame is being rendered in case it had been changed.

I will attach a working app.js file which fixes the bug on ios. Also note that registering the event in the html should be removed.


[app.zip](https://github.com/mrdoob/three.js/files/436542/app.zip)
